src/include/wiredtiger_ext.h

```diff
@@ -532,6 +532,27 @@ struct __wt_extension_api {
     int (*transaction_visible)(
       WT_EXTENSION_API *wt_api, WT_SESSION *session, uint64_t transaction_id);
 
+    /*!
+     * Initialize a spinlock
```

Comment:
These functions should have their parameters documented, similarly to the functions above.